Apps/CommonData/src/ErrorHandling/ProblemDetails.cs

```diff
@@ -13,25 +13,38 @@
 //  See the License for the specific language governing permissions and
 //  limitations under the License.
 // -------------------------------------------------------------------------
-namespace HealthGateway.Common.ErrorHandling
+namespace HealthGateway.Common.Data.ErrorHandling
```

Comment:
why move this to Common.Data?

Reply:
ProblemDetails is a brand new POCO.  There was an ApiProblemDetails in HealthGateway.Common.ErrorHandling that has been removed as it is no longer needed for out of box botnet 7 problem details.

Reply:
ProblemDetail existed in .net core since 2.1 AFAIK (https://learn.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.mvc.problemdetails?view=aspnetcore-2.1)
My question here is why put the ProblemDetail code in Common.Data assembly

Reply:
ProblemDetails is there as it is used with ProblemDetailsException and ExceptionUtility, which are also there.  As for the name 'ProblemDetails', let's discuss with @sslaws as he created it. Do you suggest naming it something else?